docs/extendedusage.rst

```diff
@@ -198,6 +200,23 @@ Traits
     'The index of the first track to return'
 
 
+RAML1.0 Types
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> api.types
+    {'Person': ObjectType(name='Person', properties={'name': Property(type='string')})}
+    >>> person = api.types['Person']
+    >>> person.type
+    'object'
+    >>> person.description
+    'a Person is a type describing human beings'
+    >>> person.properties
+    {'name': Property(type='string')})
+    >>> person.validate({'foo': 'bar'})
```

Comment:
What I mean here is to use the raml spec as a data validation mean
validate() takes an object and raise ValidationError if the object is not matching the type specified by the raml.

The background here is that my team is using http://python-eve.org/ in order to implement simple CRUD microservices backends.
python-eve is using Cerberus: http://python-eve.org/validation.html as its default data validation framework, but for me it would make a lot of sense to use raml and ramlfication instead. python-eve is already very modular in that matter.

Obviously this part is more for medium term